### PR TITLE
[9.x] Adding PHP DocBlock @mixin.

### PR DESCRIPTION
**Pull request by adideas**
The mixin DocBlock allows you to signal that all properties and methods of another class are available on the class where the mixin is applied upon.
### I think this will help a lot in learning or using Laravel!

```php
namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class Thread extends Model
{

}
```

Before: 
alt text

After:
alt text
alt text
alt text

### I think this will help a lot in learning or using Laravel!

### CODE REVIEW

**Comment by adideas** [comment]
I tested in PhpStorm.

**Comment by ankurk91** [comment]
https://github.com/barryvdh/laravel-ide-helper

**Comment by adideas** [comment]
What's the point of adding an additional dependency?  in a large laravel project this doesn't work.  this pr will tell the editor a bit about what methods exist.

**Comment by adideas** [comment]
on click I will get to the ide helper and not to the vendor

File: src/Illuminate/Database/Eloquent/Model.php [hunk 1]
```diff
@@ -23,6 +23,9 @@
 use JsonSerializable;
 use LogicException;
 
+/**
+ * @mixin Builder
```
Comment (rodrigopedra) [inline src/Illuminate/Database/Eloquent/Model.php:27]: If you search for other `@mixin` usage throughout Laravel codebase, you will see every time a Full Qualified Class Name is used.

So I would change this to:

~~~php
/**
 * @mixin \Illuminate\Database\Eloquent\Builder
~~~

**Pull request closed without merging**